ruby/lib/grpclb/server.rb

```diff
@@ -0,0 +1,37 @@
+require 'grpclb/internal/load_meter'
+require 'grpclb/internal/load_report_service'
+
+class Grpclb::Server < GRPC::RpcServer
+  def initialize(*args)
+    @meter = Grpclb::Internal::LoadMeter.new
```

Comment:
instead of implementing your own meter, can't you use something like https://github.com/grpc/grpc/blob/master/src/ruby/lib/grpc/generic/rpc_server.rb#L43?

Reply:
@dim I'd love to, but `GRPC::RpcServer` `@pool` is private, has no accessors.